_posts/2016-04-23-front-end-web-developer-ashford.md

```diff
@@ -0,0 +1,49 @@
+---
+layout:         post
+title:          "Front-end Web Developer Ashford"
+date:           2016-04-23
+author_id:      "luke_frake"
+description:    "Space Between are looking to recruite a Front-end Developer to join their growing team in Ashford Kent"
+categories:     space between
+banner:         "/assets/images/blog/macbook.jpg"
+redirect_from:
+  - space/between/2016/02/08/looking-for-a-ui-designer.html
+  - space/between/2016/02/08/looking-for-a-front-end-developer.html
+---
+
+### Salary: £18,000 - £25,000
+
+### Location: Ashford, Kent
+
+We're looking for knowledgeable developers to work with us in our office in Ashford, developing web solutions using modern web technologies.
+
+This role will see you leading the technical aspects of a project with our friendly, expert team. We work hard and keep our clients happy, but the atmosphere is positive and proactive. Our aim is to give you the resources you need to write and deploy the best code that you can.
```

Comment:
clients happy, and maintain a positive and proactive atmosphere. OR clients happy, whilst/still maintaining a positive and proactive atmosphere.